ddl/ddl.go

```diff
@@ -199,25 +198,25 @@ func setColumnFlagWithConstraint(colMap map[string]*column.Col, v *coldef.TableC
 		for _, key := range v.Keys {
 			c, ok := colMap[strings.ToLower(key.ColumnName)]
 			if !ok {
-				// TODO: table constraint on unknown column
+				// TODO: table constraint on unknown column.
 				continue
 			}
 			c.Flag |= mysql.PriKeyFlag
-			// primary key can not be NULL
+			// primary key can not be NULL.
```

Comment:
So many lowercase comments are inner functions, i have to keep them stay the same:(